integration-libs/digital-payments/schematics/constants.ts

```diff
@@ -1,6 +1,11 @@
+import { SPARTACUS_DIGITAL_PAYMENTS } from '@spartacus/schematics';
+
 export const DIGITAL_PAYMENTS_FOLDER_NAME = 'digital-payments';
 export const DIGITAL_PAYMENTS_MODULE_NAME = 'DigitalPayments';
 export const DIGITAL_PAYMENTS_MODULE = 'DigitalPaymentsModule';
 export const DIGITAL_PAYMENTS_TRANSLATIONS = 'dpTranslations';
 export const DIGITAL_PAYMENTS_TRANSLATION_CHUNKS_CONFIG =
   'dpTranslationChunksConfig';
+export const SPARTACUS_DIGITAL_PAYMENTS_ASSETS = `${SPARTACUS_DIGITAL_PAYMENTS}/assets`;
+
+export const CHECKOUT_BASE_FEATURE_NAME_CONSTANT = 'CHECKOUT_FEATURE';
```

Comment:
Let's use only one constant.
I suggest to move the one from `feature-libs/checkout/schematics/constants.ts` to `projects/schematics/src/shared/constants.ts` and use it in both feature libs.